coalib/settings/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Has Setting, Section class; indexing tools; input checks; loads bears, files.
```

Comment:
It must be prose. High level summary. See pep 257. It is hard .. ;)

Reply:
if it is *prose* then it is not detailed

Reply:
The first line should not be detailed.  It is a summary.

After the blank line, you add as many lines of prose as you want, where it can be quite detailed.

Reply:
ok I get you